src/components/Input/Input.jsx

```diff
@@ -3,15 +3,14 @@ import "./Input.css";
 
 const Input = (props) => {
   return (
-    <div className="input">
-      <Button
-        onClick={props.onClick}
-        iconName="plus"
-        variant="primary"
-        className="input__button"
+    <form onSubmit={props.onSubmit} className="input">
+      <Button iconName="plus" variant="primary" className="input__button" />
```

Comment:
in a form is good practice to set `type="submit"` inside the button that should trigger the submit